app/presenters/gobierto_data/dataset_presenter.rb

```diff
@@ -0,0 +1,76 @@
+module GobiertoData
+  class DatasetPresenter
+    include ActionView::Helpers::UrlHelper
+
+    attr_reader :site
+
+    def initialize(site)
+      @site = site
+    end
+
+    def build_catalog
+      catalog = {
+        identifier_uri: url_helpers.gobierto_data_root_url(host: site.domain),
+        title:          "dcat catalog for #{site}",
+        description:    "this is catalog published by #{@catalog_identifier_uri} which contains datasets",
+        issued:         site.created_at,
+        modified:       GobiertoData::Dataset.where(site_id: site.id).maximum(:created_at) || site.created_at,
+        languages:      site["configuration_data"]["available_locales"],
+        homepage:       url_helpers.gobierto_data_root_url(host: site.domain),
+        license_url:    "https://opendatacommons.org/licenses/odbl/",
```

Comment:
Alvaro said no license, right? Is it mandatory?

Reply:
I'm not sure, until I know from now:
DCAT don't have required, recommended, optional 
DCAT-AP it have (not 100% sure)

So I'll remove license at dataset level